### DISCUSSION TITLE
We should discuss discussions

### DISCUSSION THREAD

**Discussion by JayBeavers** [ATMakersOrg/ATDeviceLib#2] [General]
Can we pin this?

**Comment by JayBeavers** [comment]
Please not Facebook :-). They decide what we should and shouldn't see in the discussion.

**Reply by ATMakersBill** [reply]
As I said in the stream, Facebook is a necessary evil for the organization.  I really do dislike using it, but the disability support groups live there.  In part that's because their site works for accessible browsers (dragon, JAWS, etc.) and in part it's where they find each other and share what we've done.

So, the requests will come from there, the success stories have to be posted there (along w/YouTube & ATMakers.org).  However, the coordination of board design & development could be done here

**Reply by JayBeavers** [reply]
Oh, completely understand.  I was meaning for engineering discussions, Facebook is a poor forum because topics and replies get filtered and hidden based on Facebook's algorithm which determines who should see what when.

**Reply by JayBeavers** [reply]
Also, Github is terribly intimidating for "mere mortals".  I've tried pointing non-engineers to a Github page before.  Not a successful idea :-)

**Reply by pokergeist** [reply]
I see that we need three things:

1. Capture user/maker requests made via Facebook.
2. Move requests from a Requests list to a Working list as library authors take on a request.
3. Once a part is added to the library, update a catalog of available parts broken into categories.

I was hoping GitHub had a drag and drop list manager like Trello, but no. So I'll suggest using the repo Wiki for all three objectives. One page for the lists (or tables) and an index and category topic pages for devices. All very searchable.  I'd be glad to start on that.

(AKA John Jordan)

**Comment by ATMakersBill** [comment]
John,
I don't want to track user requests in this repository - Yahnatan keeps
track of those and we need to keep it where the requests come from.  This
repository is meant to be just for PCB components.  Github does have a
Projects feature that's similar to Kanban systems, but I don't think we
need that to collect and disseminate PCB parts

On Thu, Mar 10, 2022 at 5:40 PM John J. ***@***.***> wrote:

> I see that we need three things:
>
>    1. Capture user/maker requests made via Facebook.
>    2. Move requests from a Requests list to a Working list as library
>    authors take on a request.
>    3. Once a part is added to the library, update a catalog of available
>    parts broken into categories.
>
> I was hoping GitHub had a drag and drop list manager like Trello, but no.
> So I'll suggest using the repo Wiki for all three objectives. One page for
> the lists (or tables) and an index and category topic pages for devices.
> All very searchable. I'd be glad to start on that.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ATMakersOrg/ATDeviceLib/discussions/2#discussioncomment-2335211>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH2UJUV4LLKVWUJAAALW5WDU7J237ANCNFSM5P4L42OA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Bill Binko
***@***.***

"Helping Makers Help Others"
Join us at http://atmakers.org

**Reply by pokergeist** [reply]
Bill,
I agree, most Kanban systems are _way_ too structured for my needs, and I am NOT in favor of using the GH Projects resource. I was just looking for a simple focal point where we can pull something from the ToDo list to work on and later enter it as an available part.

That's why I suggested the Wiki, which could still be used as a catalog of sorts - way  better than a spreadsheet in my opinion. We could also list parts available from our private libraries that we think might be of value.

Anyway, I'll sit tight until I hear more. Thanks.

**Reply by pokergeist** [reply]
Here's a sample of what I'm proposing: https://github.com/pokergeist/nop/wiki/ATDeviceLib

**Comment by ATMakersBill** [comment]
So I’m good with listing out the components that way but I think we should
offer two options: if the parts are licensed in a way that lets them be in
our library we should copy them there. If not we should point to the
official source. So “Ultra Librarian” or Digi-Key etc.
does that make sense?
On Fri, Mar 11, 2022 at 9:46 AM John J. ***@***.***> wrote:

> Here's a sample of what I'm proposing:
> https://github.com/pokergeist/nop/wiki/ATDeviceLib
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ATMakersOrg/ATDeviceLib/discussions/2#discussioncomment-2339726>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH2UJUQWOQBZ3Z3L54G4X2LU7NMDNANCNFSM5P4L42OA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Bill Binko
***@***.***

"Helping Makers Help Others"
Join us at http://atmakers.org
